backend/escalation_engine.py

```
@@ -0,0 +1,286 @@
+            audit_log = EscalationAudit(
+                grievance_id=grievance.id,
+                previous_authority=previous_authority,
+                new_authority=grievance.assigned_authority,
+                reason=reason,
+                notes=notes
+            )
+
+            db.add(audit_log)
+            db.commit()
+
+            return True
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)